File: deps/v8/tools/run-tests.py
#!/usr/bin/env python3
#
# Copyright 2017 the V8 project authors. All rights reserved.
# Use of this source code is governed by a BSD-style license that can be
# found in the LICENSE file.


from __future__ import absolute_import
import sys

from testrunner import standard_runner


if __name__ == "__main__":
  sys.exit(standard_runner.StandardTestRunner().execute())

File: deps/v8/test/inspector_protocol_parser_test/RunTests.cpp
// Copyright 2016 the V8 project authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

#include "testing/gmock/include/gmock/gmock.h"

namespace {

class InspectorProtocolTestEnvironment final : public ::testing::Environment {};

}  // namespace

int main(int argc, char** argv) {
  testing::InitGoogleMock(&argc, argv);
  testing::AddGlobalTestEnvironment(new InspectorProtocolTestEnvironment);
  return RUN_ALL_TESTS();
}
